### DISCUSSION TITLE
GitHub Action "uses" dependencies aren't parsed when there is a space before the colon

### DISCUSSION THREAD

**Discussion by RBusarow** [renovatebot/renovate#27088] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.173.2

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

This workflow file is accepted by GitHub.  The `actions/checkout@v3` is out of date, but Renovate doesn't mention it in the logs and doesn't create a pull request.

```yaml
# .github/workflows/canary.yml
name: Canary

on:
  pull_request:

jobs:
  unit-tests:
    runs-on: ubuntu-latest
    steps:
      - uses : actions/checkout@v3
```

If I remove the space before the colon, Renovate will pick up the dependency and make a pull request.

repro project/branch: https://github.com/rickbusarow/Renovate-Config-Test/tree/renovate-space-repro

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: packageFiles with updates (repository=rickbusarow/Renovate-Config-Test, baseBranch=main)
       "config": {
         "github-actions": [
           {
             "deps": [
               {
                 "depName": "ubuntu",
                 "currentValue": "latest",
                 "replaceString": "ubuntu-latest",
                 "depType": "github-runner",
                 "datasource": "github-runners",
                 "autoReplaceStringTemplate": "{{depName}}-{{newValue}}",
                 "skipReason": "invalid-version",
                 "updates": [],
                 "packageName": "ubuntu"
               }
             ],
             "packageFile": ".github/workflows/canary.yml"
           },
           {
             "deps": [
               {
                 "depName": "ubuntu",
                 "currentValue": "latest",
                 "replaceString": "ubuntu-latest",
                 "depType": "github-runner",
                 "datasource": "github-runners",
                 "autoReplaceStringTemplate": "{{depName}}-{{newValue}}",
                 "skipReason": "invalid-version",
                 "updates": [],
                 "packageName": "ubuntu"
               }
             ],
             "packageFile": ".github/workflows/renovate.yml"
           }
         ]
       }
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
Fix: https://github.com/renovatebot/renovate/pull/27170
